definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -6621,13 +6600,6 @@
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "CRU"
   },
-  {
```

Comment:
are these deletion part of the stories?

Reply:
Not required for this story, but were causing import error. Hemanth thinks it's recent validation changes. The field was never actually used and was added at the request of the business to utilise the newly delivered CCD concatenation feature.

Reply:
sure, added it back in